### DISCUSSION TITLE
Risky but respectable

### DISCUSSION THREAD

**Discussion by OMetaVR** [familiar/Bedwars-Bot#5] [Ideas]
I just have to ask, so you coded a whole bot out of a macro mod, cool stuff, but then you post it to get it banned and then showed that you'd been using it for a month and a half with the account name on screen and everything? I very much hope that account is an alt of yours, and not your main account. Not that I believe anyone would care enough to report your account but that is worrying.

**Comment by iHeq** [comment]
tarded

**Comment by IDoTheHax** [comment]
It's a macro not real working bot like I thought, I'm currently working on training a bot to play the game successfully using an AI model but this would be more practical for scenario pvp training

**Comment by familiar** [comment]
My main is perm banned, it's just an alt.
